README.md

```diff
@@ -104,7 +104,7 @@ Places each given node. It is assumed that the remaining nodes in the graph alre
 
 ```instance.packComponents(components)```
 
-Packs components of a disconnected graph.
+Packs components of a disconnected graph. Layout is done in a way that it preserves the center of the  [bounding rectangle](occupiedRectangleReal) of components. 
```

Comment:
Let's use 'Packing' instead of 'Layout'. I didn't understand the part with 'bounding rectangle' and 'occupiedRectangleReal', it creates a link on 'bounding rectangle' that goes nowhere.

Reply:
I was intended to refer to the wikipedia page about [bounding rectangles](https://en.wikipedia.org/wiki/Minimum_bounding_rectangle). Now changed it accordingly